server/@types/welcome/index.d.ts

```diff
@@ -23,6 +23,14 @@ declare module 'welcome' {
     TRANS = 'TRANS',
   }
 
+  type PotentialMatch = {
+    firstName: string
+    lastName: string
+    dateOfBirth: string
+    prisonNumber?: string
+    pncNumber?: string
+  }
+
```

Comment:
doesn't need to be exported on its own as will always be a part of the Movement type